src/patternfly/components/OptionsMenu/examples/options-menu.md

```diff
@@ -110,32 +110,29 @@ import './options-menu.css'
 
 ### Plain with text
 ```hbs
-{{#> options-menu id="options-menu-disabled-text-example" options-menu--IsText="true" options-menu-toggle--IsDisabled="true"}}
-  {{#> options-menu-toggle options-menu-toggle--type="div" options-menu-toggle--modifier="pf-m-plain"}}
+{{#> options-menu id="options-menu-disabled-text-example" options-menu--IsText="true" options-menu-toggle--IsDisabled="true" options-menu--HasToggleIcon="true"}}
```

Comment:
> options-menu--IsText="true"

There is some logic in `options-menu-toggle.hbs` that assumes this toggle is a `div` and removes necessary attributes if `--IsText` is set to `true`

https://github.com/patternfly/patternfly/blob/9f8b5abbfb9af04cc243b5340409ad5c8bf7a868/src/patternfly/components/OptionsMenu/options-menu-toggle.hbs#L2-L10